Jenkinsfile_parameterized

```diff
@@ -0,0 +1,20 @@
+#!groovy
+@Library("Infrastructure") _
+
+properties([
+		parameters([
+			string(name: 'PRODUCT_NAME', defaultValue: 'platform', description: ''),
+			string(name: 'APP', defaultValue: 'job-scheduler', description: ''),
+			string(name: 'TYPE', defaultValue: 'java', description: ''),
+			string(name: 'ENVIRONMENT', defaultValue: 'sandbox', description: 'Environment where code should be build and deployed'),
+			choice(name: 'SUBSCRIPTION', choices: 'sandbox\nnonprod\nprod', description: 'Azure subscriptions available to build in')
```

Comment:
probably a typo in `nonprod`

Reply:
that's a new line + nonprod. makes `\nnonprod`

Reply:
:D my bad. Out of interest, can it not be comma separated?

Reply:
looked and thought all this infra code is somewhere in our repos. but did not find. found same references e.g. [here](https://github.com/hmcts/moj-infra-pipeline/blob/8fc8b95cd3d024d40e05c982e4ab551211f24303/Jenkinsfile#L7)